doc/getting_started/index.rst

```diff
@@ -3,319 +3,372 @@
 
-- On macOS, `Homebrew disables -\\-user`_.
+      On macOS Mojave, you can manually check for updates by choosing
+      System Preferences from the Apple menu, then click Software Update (and
+      click Update Now if there are). For other macOS versions, see the
+      `Update macOS topic in Apple support
+      <https://support.apple.com/en-us/HT201541>`_.
 
-- On Windows, see the Installing Packages information on ``--user`` if you
-  require using this option.
+   .. group-tab:: Windows
 
-On all operating systems, the ``-U`` flag installs or updates the package if the
-package is already installed locally but a more recent version is available. It
-is good practice to use this flag if the latest version of a package is
-required.
+      On Windows, you can manually check for updates by selecting Start  > Settings  >
+      Update & Security  > Windows Update, and then select Check for updates.
+      If updates are available, install them.
 
 .. _install-required-tools:
 
-Install the required tools
-===========================
+Install dependencies
+====================
 
-Follow an operating system specific guide, then come back to this page.
+Next, use your OS package manager to install required OS-provided
+support tools.  For example, Python 3 and its package manager, pip, are
+used extensively by Zephyr for installing and running scripts used to
+compile, build, and run Zephyr applications.
 
-.. toctree::
-   :maxdepth: 1
+.. tabs::
 
-   Linux <installation_linux.rst>
-   macOS <installation_mac.rst>
-   Windows <installation_win.rst>
+   .. group-tab:: Linux
 
-.. _get_the_code:
+      #. Install Linux OS-provided support tools:
 
-Get the source code
-*******************
+         .. code-block:: bash
 
-Zephyr's multi-purpose :ref:`west <west>` tool lets you easily get the Zephyr
-project repositories. (While it's easiest to develop with Zephyr using west,
-we also have :ref:`documentation for doing without it <no-west>`.)
+            sudo apt install --no-install-recommends git cmake ninja-build gperf \
+              ccache dfu-util device-tree-compiler wget \
+              python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
+              make gcc gcc-multilib
 
-Install west
-============
+      #. Install needed Python packages:
 
-First, install ``west`` using ``pip3``:
+         .. code-block:: bash
 
-.. code-block:: console
+            pip3 install --user -U west
+            pip3 install --user -r https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/master/scripts/requirements.txt
+            echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+            source ~/.bashrc
 
-   # Linux
-   pip3 install --user -U west
+         The ``--user`` option puts installed packages into your
+         ``~/.local/bin folder``.  It's very important you permanently update the
+         PATH environment variable with the location of these packages (in your
+         .bashrc profile, as shown).
```

Comment:
Please avoid the word "profile" when referring to `.bashrc` considering the _alternatives_ to `.bashrc` are named `.profile`, `.bash_profile` and what not.

Has the .bashrc command above been fully tested when connecting remotely/graphically/scriptically/using zsh... ? (hi @carlescufi!)

Linux environment variables are a non-portable disaster: https://superuser.com/questions/183870/difference-between-bashrc-and-bash-profile/1452523#1452523
Yet another reason to rename the "Linux" tab in this page to "Ubuntu 12.34".

Reply:
I misread this sentence to mean that you should regularly update the PATH environment variable. Maybe rephrase to something like "Adding the PATH specification to your .bashrc file ensures that the change is permanent."